btr-web/btr-main-app/components/bcros/CompletingParty.vue

```diff
@@ -0,0 +1,161 @@
+<script setup lang="ts">
+import { getDefaultInputFormCompletingParty } from '~/utils/omit-schema/defaults'
+import { CompletingPartySchemaType, CompletingPartySchema } from '~/utils/omit-schema/definitions'
+import { CompletingIndividualTypeE, CompletingIndividualTranslationsE } from '~/enums/omit/completing-individual-type-e'
+const t = useNuxtApp().$i18n.t
+
+const model = defineModel({ type: Object, default: getDefaultInputFormCompletingParty() })
+
+defineProps({
+  editMode: {
+    type: Boolean,
+    required: false,
+    default: true
+  }
+})
+
+const completingPartyForm = ref()
+
+const validate = async function () {
+  await completingPartyForm.value.validate(null, { silent: true })
+}
+
+defineExpose({
+  validate,
+  completingPartyForm
+})
+
+const individualTypeOptions = []
+for (const key in CompletingIndividualTypeE) {
+  individualTypeOptions.push({ value: key, label: t(CompletingIndividualTranslationsE[key]) })
+}
+
+function hasErrors (sectionErrorPaths: string[]): boolean {
+  if (!completingPartyForm.value?.errors) {
+    return false
+  }
+  for (const errorPath of sectionErrorPaths) {
+    const errors = completingPartyForm.value.errors
+    if (!errors || errors.length === 0) {
+      return false
+    }
+
+    const hasErrors = errors.filter(errObj => errObj.path.includes(errorPath)).length > 0
+    if (hasErrors) {
+      return true
+    }
+  }
+
+  return false
+}
+
+const formSchema: CompletingPartySchemaType = CompletingPartySchema
+
+</script>
+
+<template>
+  <UForm
+    ref="completingPartyForm"
+    :schema="formSchema"
+    :state="model"
+    class="w-full"
+  >
+    <div class="mt-2 w-full border-t border-solid pt-4">
+      <p>
+        {{ $t('labels.requestOmit.completingPartyIntro') }}
+      </p>
+      <BcrosSection
```

Comment:
can you add a 'padded' prop for the section and set it to false for all the bcros sections in here? The px-8 padding is putting everything out of line with the title etc. like the design

Reply:
I added padded-x and padded-y as padded-x is what we want to drop here, not padded-y but for completeness I added padded-y as well